packages/abstractions/package.json

```diff
@@ -36,6 +36,7 @@
   },
   "dependencies": {
     "@opentelemetry/api": "^1.2.0",
+    "guid-typescript": "^1.0.9",
```

Comment:
Just to confirm, shouldn't we bump up the package versions here so that this builds after release?

Reply:
yes we should, but only at the last minute since the tags will be what gets published